src/main/java/frc/robot/RobotContainer.java

```diff
@@ -122,6 +122,8 @@ private void configureBindings()
     // Schedule `ExampleCommand` when `exampleCondition` changes to `true`
 
     driverXbox.a().onTrue((Commands.runOnce(drivebase::zeroGyro)));
+    driverXbox.b().onTrue(MechanismCommands.absoluteEncoderPosition(drivebase));
```

Comment:
We could comment out the code but I think we shouldn't delete the stuff related to encoder offsets